web/templates/rename_topic.hbs

```diff
@@ -0,0 +1,20 @@
+<form class="new-style message-move-form" id="rename_topic_form">
+    <p>{{t "Rename topic to:" }}</p>
+    <div class="topic_stream_edit_header">
+        <input name="new_topic_name" type="text" class="inline_topic_edit modal_text_input" autocomplete="off" value="{{topic_name}}" />
+        <input name="old_topic_name" type="hidden" class="inline_topic_edit" value="{{topic_name}}" />
+        <input name="current_stream_id" type="hidden" value="{{current_stream_id}}" />
+        <div class="topic_move_breadcrumb_messages">
+            <label class="checkbox">
+                <input class="send_notification_to_new_thread" name="send_notification_to_new_thread" type="checkbox" {{#if notify_new_thread}}checked="checked"{{/if}} />
+                <span></span>
+                {{t "Send automated notice to new topic" }}
+            </label>
+            <label class="checkbox">
+                <input class="send_notification_to_old_thread" name="send_notification_to_old_thread" type="checkbox" {{#if notify_old_thread}}checked="checked"{{/if}} />
+                <span></span>
+                {{t "Send automated notice to old topic" }}
+            </label>
+        </div>
+    </div>
+</form>
```

Comment:
What's the thinking on doing a separate template vs a couple conditionals within the template? I think, for example, we don't actually need a separate ID for this form from the other one, since both can't be visible at the same time anyway.